spec.emu

```diff
@@ -275,12 +276,99 @@
       <emu-note>That means that "A" (`LATIN CAPITAL LETTER A`) measures as 1 code unit, and "🔥" (`FIRE`) measures as 2 code units. Source maps for other content types may diverge from this.</emu-note>
     </dd>
   </dl>
+
+  <emu-clause id="sec-vlq-grammar">
+    <h1>base64 VLQ grammar</h1>
+    <p>A base64-encoded variable-length quantity adheres to the following lexical grammar:</p>
+
+    <emu-grammar type="definition">
+      Vlq ::
+        VlqDigitList
+
+      VlqDigitList ::
+        DigitWithoutContinuationBit
+        DigitWithContinuationBit VlqDigitList
+
+      DigitWithoutContinuationBit :: one of
+        `A` `B` `C` `D` `E` `F` `G` `H` `I` `J` `K` `L` `M` `N` `O` `P` `Q` `R`
+        `S` `T` `U` `V` `W` `X` `Y` `Z` `a` `b` `c` `d` `e` `f`
+
+      DigitWithContinuationBit :: one of
```

Comment:
Very nitpicky, but I would find it easier to read if there was a bigger difference between the two names. Wdyt about `TerminalDigit` and `ContinuationDigit`?

Reply:
I like it! Done.